local/mib2c-conf.d/mfd-access-container-cached-defines.m2i

```diff
@@ -358,7 +358,7 @@ $example_end
 @end@ # example
 
     DEBUGMSGT(("verbose:${context}:${context}_container_load",
-               "inserted %d records\n", count));
+               "inserted %zu records\n", count));
```

Comment:
How about using NETSNMP_PRIz instead of 'z'? The 'z' format specifier is supported by glibc but not by all versions of Microsoft Visual Studio.

Reply:
Done.